src/apps/vpn/controller.cpp

```diff
@@ -528,6 +528,8 @@ void Controller::connected(const QString& pubkey,
     resetConnectedTime();
   }
 
+  QString sessionId = mozilla::glean::session::session_id.generateAndSet();
```

Comment:
If you don't use `sessionId`, don't create it.

Reply:
Same as above, it will be used in VPN-4038.